.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: python
+python:
+  - "3.6"
+install:
+  - pip install -r requirements.txt
```

Comment:
Did you test this?
Do we need to update the requirements.txt?

Reply:
The best way to test this is configure the Repo on the Travis portal, then make any push and it should "build" - currently there is no requirements.txt so we may need to add this. Configuration should be easy - see image 

![image](https://user-images.githubusercontent.com/26774878/36817155-020aa670-1c95-11e8-94b5-ab1d799bee77.png)

Reply:
Sure so you are saying "no" and "yes" OK let's do that then. We don't need to take this PR until we have those then.

Reply:
I just tested locally and integration with Travis-CI worked, just need requirements.txt and the Repo owner needs to configure it in the Travis portal